packages/react/src/_popover/Popover.tsx

```diff
@@ -81,39 +86,33 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 			// Adds distance between the reference and floating element
 			// https://floating-ui.com/docs/offset
 			offset(offsetOption),
+			// Placing shift() before flip() in the array ensures it can do its work before flip() tries to change the placement.
+			// https://floating-ui.com/docs/flip#combining-with-shift
+			shift({ padding: MIN_SIDE_GUTTER_WIDTH }),
```

Comment:
Still really hate the massive shift that can occur

Reply:
This isn't new, but it's weird. Why is the calendar nowhere near? 
![calendar-nowhere-near](https://github.com/agriculturegovau/agds-next/assets/853552/6de10ec0-bc73-4d52-99f1-c4c9922cc837)

Reply:
The date range fieldset is the anchor point. This makes more sense on desktop but not really on mobile. We can control the offset the calendar renders from the anchor but I'd have to look into a neat way to do this. Might be another ticket for this one but it's defs something worth doing, IMO.

Reply:
>Still really hate the massive shift that can occur

Does this happen on mobile? Or just mobile-like desktop screens? I can't replicate this on my mobile device.

Reply:
> Does this happen on mobile? Or just mobile-like desktop screens? I can't replicate this on my mobile device.

Yep, I can reproduce on my phone. Also, for some reason, focus gets moved to the input briefly, so the keyboard appears which reduces the available space and puts the calendar at the top, and then it closes so moves it back below. I'll upload a video separately

Reply:
https://github.com/agriculturegovau/agds-next/assets/853552/64525eac-59bf-4920-b812-8f9cb319e51d

Reply:
I can't produce the focusing on button oress any more, however, if the field has focus (by clicking the label) and then you click the calendar, you get a similar jump in position as the keyboard dismisses. Not sure if there's anything we can do about that though